### ISSUE TITLE
Feature Request: Custom headers on cloning landing page

### ISSUE THREAD

**Issue by W1R3D-Code** [gophish/gophish#772]
It would be really useful if there was an option to paste in the headers from an authenticated session when cloning a page, so it could clone pages that are behind access control i.e. an update details / change password page in a users account settings.

It wouldn't need to be anything fancy, just parse and repeat :) even if it was in the settings somewhere to be used on every cloning request (if that makes the UI change any simpler...at the expense of making it persisted)

**Comment by jordan-wright** [comment]
Yeah, I bet we could do that. It'll probably be a low-priority item for my personal queue of features to knock out since I have other big changes to make but I think there's room in our "Import Site" modal to add at least an expandable "Custom Headers" option.

**Comment by ponchohoncho** [comment]
For what it's worth, I'm enjoying this Chrome plugin for cloning pages: https://chrome.google.com/webstore/detail/singlefile/mpiodijhokgodhhofbcjdecpffjipkle?hl=en

It takes every asset and embeds it in the HTML, including the images. It'll sometimes require a little massaging, and the generated HTML files are definitely on the heavy side, but I've done OK with it.

**Comment by W1R3D-Code** [comment]
Ah nice one! I ended up write a console app to do the same thing in a much
more convoluted way ^.^
